clash-prelude/src/Clash/Explicit/BlockRam.hs

```diff
@@ -763,30 +774,42 @@ blockRam
   => Clock dom
   -- ^ 'Clock' to synchronize to
   -> Enable dom
-  -- ^ Global enable
+  -- ^ 'Enable' line
   -> Vec n a
-  -- ^ Initial content of the BRAM, also determines the size, @n@, of the BRAM.
+  -- ^ Initial content of the BRAM, also determines the size, @n@, of the BRAM
```

Comment:
I would change "BRAM" to "block RAM" in documentation comments

Reply:
This comes up quite a bit, so I'll only comment it here for the first one

Reply:
Leaving it as is as discussed.